gdal/alg/gdal_rpc.cpp

```diff
@@ -886,6 +891,16 @@ void *GDALCreateRPCTransformer( GDALRPCInfo *psRPCInfo, int bReversed,
         psTransform->dfDEMMissingValue = CPLAtof(pszDEMMissingValue);
     }
 
+/* -------------------------------------------------------------------- */
+/*                        The DEM SRS override                          */
+/* -------------------------------------------------------------------- */
+    const char *pszDEMSRS =
+        CSLFetchNameValue(papszOptions, "RPC_DEM_SRS");
+    if ( pszDEMSRS != nullptr )
+    {
+        psTransform->pszDEMSRS = CPLStrdup(pszDEMSRS);
```

Comment:
Needs:
- to be free'd in GDALDestroyRPCTransformer()
- serialized in GDALSerializeRPCTransformer()
- deserialized in GDALDeserializeRPCTransformer()

To check the correct behaviour of serialization/deserialization, you can gdalwarp to a VRT file and then gdal_translate that VRT file